<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>footswitch</class>
 <widget class="QDialog" name="footswitch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(18, 78, 102);


</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-170</x>
     <y>150</y>
     <width>881</width>
     <height>661</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
border:none;
image: url(:/images/footimges.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>120</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:30pt &quot;Ubuntu&quot;;
border-radius:20px;border:2px solid black;
background-color: rgb(0, 0, 0);
background-color: rgb(98, 160, 234);font:bold;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>310</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:30pt &quot;Ubuntu&quot;;
background-color:#00B2FF;
color: rgb(0, 0, 0);
border-radius:20px;border:2px solid black;font:bold;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>530</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 30pt &quot;Ubuntu&quot;;
background-color: #007BFF; /* Attractive Blue for second position */
color: rgb(0, 0, 0);
border-radius: 20px;
border: 2px solid black;
font:bold;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>770</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 30pt &quot;Ubuntu&quot;;
background-color: #0056b3; /* Darker Blue */
color: rgb(0, 0, 0);
border-radius: 20px;
border: 2px solid black;
font:bold;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="But_zero">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>80</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background-color:transparent;
image: url(:/images/uarrow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="But_zerodown">
   <property name="geometry">
    <rect>
     <x>1690</x>
     <y>80</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background-color:transparent;
image: url(:/images/darrow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="But_onedown">
   <property name="geometry">
    <rect>
     <x>1690</x>
     <y>270</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background-color:transparent;
image: url(:/images/darrow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="But_oneup">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>280</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background-color:transparent;
image: url(:/images/uarrow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="But_twodown">
   <property name="geometry">
    <rect>
     <x>1690</x>
     <y>490</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background-color:transparent;
image: url(:/images/darrow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="But_twoup">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>500</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background-color:transparent;
image: url(:/images/uarrow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="But_threedown">
   <property name="geometry">
    <rect>
     <x>1690</x>
     <y>730</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background-color:transparent;
image: url(:/images/darrow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="But_threeup">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>730</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background-color:transparent;
image: url(:/images/uarrow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>470</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/circle.png);
background-color:transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="But_save">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>950</y>
     <width>501</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:35pt;
color: rgb(255, 255, 255);
border:1px solid white;
background-color: rgb(12, 40, 82);</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="But_topleft">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { text-align: center; font-size: 25pt; 
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius:20px;
border:2px solid skyblue;
}</string>
   </property>
   <property name="text">
    <string>Top Left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="But_topright">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>660</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { text-align:center; font-size:25pt; 
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius:20px;
border:2px solid skyblue;
}</string>
   </property>
   <property name="text">
    <string>Top Right</string>
   </property>
  </widget>
  <widget class="QPushButton" name="But_bottomleft">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { text-align:center; font-size: 25pt; 
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius:25px;
border:2px solid skyblue;
}</string>
   </property>
   <property name="text">
    <string>Bottom left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="But_bottomright">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>670</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { text-align:center; font-size: 25pt; 
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius:20px;
border:2px solid skyblue;
}</string>
   </property>
   <property name="text">
    <string>Bottom Right</string>
   </property>
  </widget>
  <widget class="QPushButton" name="But_footswitch">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>431</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background-color: transparent;
image: url(:/images/g2332.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="LeftFoot_com">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
color: rgb(255, 255, 255);
font:20pt &quot;Ubuntu&quot;;
background-color: rgb(0, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string>Increment</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Decrement</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Reflux</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Continuous Irrigation</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Power ON/OFF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PDM</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="Right_footcom">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>780</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
color: rgb(255, 255, 255);
font:20pt &quot;Ubuntu&quot;;
background-color: rgb(0, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string>Increment</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Decrement</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Reflux</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Continuous Irrigation</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Power ON/OFF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PDM</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="Bottom_leftcom">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
color: rgb(255, 255, 255);
font:20pt &quot;Ubuntu&quot;;
background-color: rgb(0, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string>Increment</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Decrement</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Reflux</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Continuous Irrigation</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Power ON/OFF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PDM</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="Bottom_rightcom">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>790</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
color: rgb(255, 255, 255);
font:20pt &quot;Ubuntu&quot;;
background-color: rgb(0, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string>Increment</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Decrement</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Reflux</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Continuous Irrigation</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Power ON/OFF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PDM</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>50</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:35px;background-color:transparent;
color:white;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>0 Position</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>230</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:35px;background-color:transparent;
color:white;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>1 Position</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>440</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:35px;background-color:transparent;
color:white;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>2 Position</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>690</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:35px;background-color:transparent;
color:white;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>3 Position</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
image: url(:/images/settings.png);
background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="But_save_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>950</y>
     <width>451</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:35pt;
color: rgb(255, 255, 255);
border:1px solid white;
background-color: rgb(12, 40, 82);</string>
   </property>
   <property name="text">
    <string>Load Default</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>290</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/uparrow.png);
border:none;
background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/uparrow.png);
border:none;
background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>590</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
image: url(:/images/downarrow.png);
background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>590</y>
     <width>61</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
image: url(:/images/downarrow.png);
background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>950</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/singletick.png);
background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>950</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/singletick.png);
background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>lineEdit_4</zorder>
  <zorder>But_zero</zorder>
  <zorder>But_zerodown</zorder>
  <zorder>But_onedown</zorder>
  <zorder>But_oneup</zorder>
  <zorder>But_twodown</zorder>
  <zorder>But_twoup</zorder>
  <zorder>But_threedown</zorder>
  <zorder>But_threeup</zorder>
  <zorder>But_save</zorder>
  <zorder>But_topleft</zorder>
  <zorder>But_topright</zorder>
  <zorder>But_bottomleft</zorder>
  <zorder>But_bottomright</zorder>
  <zorder>But_footswitch</zorder>
  <zorder>LeftFoot_com</zorder>
  <zorder>Right_footcom</zorder>
  <zorder>Bottom_leftcom</zorder>
  <zorder>Bottom_rightcom</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>pushButton</zorder>
  <zorder>But_save_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>label_2</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
